types/test/sequelize.ts

```diff
@@ -65,3 +65,13 @@ sequelize.query('SELECT * FROM `user`', { type: QueryTypes.RAW }).then(result =>
   const arraysOnly = (a: any[]) => a;
   arraysOnly(data);
 });
+
+sequelize
+  .query("SELECT COUNT(1) as count FROM `user`", {
+    type: QueryTypes.SELECT,
+    plain: true
+  })
+  .then(result => {
+    const oneRowOnly = (a: any) => a;
+    oneRowOnly(result);
```

Comment:
I don't think this is really testing anything, because it is passing `result` into an parameter that expects `any`, so it will certainly work anyway.

Reply:
Yeah, was a bit confused by the way the other tests were written 😅